tests/Fixtures/TestBundle/Entity/User.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the PostmanGeneratorBundle package.
+ *
+ * (c) Vincent Chalamon <vincentchalamon@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace PostmanGeneratorBundle\Tests\Fixtures\TestBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ *
+ * @author Yann Prou <yann@les-tilleuls.coop>
+ */
+class User
+{
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer")
+     * @ORM\Id
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    protected $id;
```

Comment:
actually the property could be made `public` instead? There is no use for getters/setters for this test class is there?